packages/web/package.json

```diff
@@ -120,6 +121,7 @@
     "nodemon": "^2.0.2",
     "postcss-scss": "^2.0.0",
     "react-native-svg": "^9.13.6",
+    "typescript": "^3.8.3",
```

Comment:
wont this risk us not using the same version of typescript on all projects?

Reply:
well, this is the case right now anyway and that will not change. If one projects added typescript every project can use it without declaring it in its own `package.json`. It's very hard to do dependancy management cross packages. Because if the package containing typescript reference is updated it might break something else somewhere else. You never know.